sdk/core/azure-core/inc/azure/core/http/http.hpp

```diff
@@ -52,8 +52,8 @@ namespace Azure { namespace Core { namespace Http {
   /**
    * @brief HTTP transport layer error.
    */
-  struct TransportException : public Azure::Core::RequestFailedException
-  {
+  class TransportException : public Azure::Core::RequestFailedException {
```

Comment:
Why do we have curly brace for classes on the same line, but on new line for structs?

Reply:
cc @CaseyCarter, @BillyONeal - thoughts? I think we should be consistent for both, here.

Reply:
I don't see it in guidelines (we should add it either way), but I vaguely recall we discussed where to put braces/new lines back in April. Does anyone remember where we landing?

Reply:
@antkmsft may be know?

Reply:
I 100% agree they should be consistent, and have no technical arguments to support either brace placement.

Reply:
Sounds good. Filed an issue to update the guidelines: https://github.com/Azure/azure-sdk/issues/1958

Let's fix it across the repo to be consistent in a separate PR along with updating the clang format rule.

Reply:
We go with the curly brace on a new line everywhere, this is a bug; This specific instance of struct/class difference can't be right regardless of what you think about the curly brace placement.